recipes/mod_application_initialization.rb

```diff
@@ -20,4 +20,7 @@
 
 include_recipe 'iis'
 
-windows_feature 'IIS-ApplicationInit'
+install_method = node['iis']['windows_feature_install_method']
+windows_feature 'IIS-ApplicationInit' do
+  install_method install_method unless install_method.nil?
```

Comment:
If we are doing this everywhere I think we should just set the attribute to the current default in the attributes file?

Reply:
my thinking here was to essentially always fallback to whatever windows_feature used as its default. That why if that resource implementation ever change, this cookbook would require no updates to match. I'm happy to not take that route though, as it would certainly make this change a lot less verbose.